packages/default/scss/treeview/_layout.scss

```diff
@@ -1,7 +1,8 @@
-@include exports("treeview/layout") {
-
-    $treeview-indent: 16px;
+$treeview-indent: 16px !default;
+$treeview-item-space-y: $button-padding-y !default;
+$treeview-item-space-x: $button-padding-x !default;
```

Comment:
In that particular case, `padding-*` seems more accurate than `space-*`, because this is the padding of an item. `spacer` / `gap` / `offset` is better when there are multiple ways to achieve a given styling, e.g. when describing the space between items or the offset of an icon, where `padding` would be too limiting.